uw-frame-components/css/buckyless/notifications.less

```diff
@@ -133,6 +133,22 @@
   font-size: 20px;
 }
 .menu-notification-link {
+  .arrow-down {
+  	width: 0; 
+  	height: 0; 
+  	border-left: 7px solid transparent;
+  	border-right: 7px solid transparent;
+  	border-top: 7px solid lighten(@color1, 20%);
+    z-index:2000;
```

Comment:
i did notice the app-header is 2000, that also seems silly. I'm going to add an issue to fix z-indexing in general